### ISSUE TITLE
Not working with Brave - Could not load manifest

### ISSUE THREAD

**Issue by shazwozza** [translate-tools/linguist#407]
CRX install method:
Downloaded .zip and .crx
Enabled developer mode in Brave extension manager.
Dragged and dropped .crx in extension manager .crx. 
Shows up in extension manager but with error message:
"This extension is not listed in the Brave Web Store and may have been added without your knowledge. "

chromium.zip file install method:
I unzipped linguist .zip file into a folder.
Click load unpacked in extension manager.
Select folder and hit ok.
Error message (below).
I also unzipped the .crx file and tried loading the folder unpacked but it came with the same error:

Error message loading unpacked:
Failed to load extension
File
~\Downloads\linguist
Error
Invalid locale file 'C:\Users\username\Downloads\linguist\_locales\gu\messages.json': Variable $translator_name$ used but not defined.
Could not load manifest.

**Comment by vitonsky** [comment]
Last error seems as duplicate of https://github.com/translate-tools/linguist/issues/403
I will fix this problem soon.

Is Guam language set as default language in your system or browser?

**Comment by shazwozza** [comment]
English is default language for system and browser.

**Comment by vitonsky** [comment]
It looks the Brave browser preloads all localization files. Is Linguist works for you in google chrome?

It seems as duplicate of #403, i will fix this when will have time to analyze all localization files and add test for it, in near 1-2 weeks

**Comment by shazwozza** [comment]
I added in Chrome by drag and dropping the .crx file with developer mode enabled.
It shows up in extension list but it cannot be enabled. 
In the description it says
"This extension is not listed in the Chrome Web Store and may have been added without your knowledge. [Learn more](https://support.google.com/chrome/?p=ui_remove_non_cws_extensions&hl=en-GB)"

**Comment by vitonsky** [comment]
@shazwozza yeah, i know about this nonsense behavior of chromium browsers, try to install from google app store: https://chrome.google.com/webstore/detail/gbefmodhlophhakmoecijeppjblibmie

**Comment by shazwozza** [comment]
If i add through that link - it says "Blocked by your organization."
Your organization blocked this file because it didn't meet a security policy.

**Comment by shazwozza** [comment]
Found a fix at least temporarily

Go to regedit:
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\BraveSoftware\Brave\ExtensionInstallAllowlist
Add String Value
Inside the "Value data:" - Copy the ID that is found in brave://extensions/ underneath the Linguist extension.
Restart browser

**Comment by hamirmahal** [comment]
This extension worked when I installed it for Brave through the Chrome web store.

```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.3 LTS
Release:	22.04
Codename:	jammy
```

**Comment by vitonsky** [comment]
Been fixed in https://github.com/translate-tools/linguist/issues/403

Thanks for your reports
